modules/gis/src/index.js

```diff
@@ -1 +1,2 @@
 export {geojsonToBinary, TEST_EXPORTS} from './lib/geojson-to-binary';
```

Comment:
`geoJsonToBinary` (capital J) for consistency?

Reply:
Yes I was thinking about that... Not sure if an issue that it's already released

Reply:
You can add always a deprecated section to avoid breaking old code.


```js
export {geoJsonToBinary} from '...

// DEPRECATED in v2.3
export {geoJsonToBinary as geojsonToBinary} from '...
```

You can also add `@deprecated` message to the typescript declaration comment.